### PR TITLE
feat: Show alert if scheduler is inactive or app version is stale

### LINKED ISSUE #70: Easier error analysis
**Banking Settings** should display a warning headline if...

- the currently installed version is older than the latest GitHub release
- the scheduler is not running (how can we reliably check that?)

### PR DESCRIPTION
**Pull request by marination**
Resolves https://github.com/alyf-de/banking/issues/70

Screenshot 2023-11-02 at 4 10 07 PM

- The scheduler being inactive is a good enough check. If the scheduler is not working due to some kind of error, the web server would also most likely go down, taking down the site
- The app version check does a basic latest release vs in-app version comparison

### CODE REVIEW

File: banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py [hunk 1]
```diff
@@ -145,3 +145,28 @@ def get_customer_portal_url() -> str:
...
+	Returns the app health.
+	"""
+	from frappe.utils.change_log import check_release_on_github
+	from frappe.utils.scheduler import is_scheduler_inactive
+
+	messages = {}
+	current_app_version = frappe.get_attr("banking.__version__")
+
+	latest_app_version = str(check_release_on_github("banking")[0])
+	if latest_app_version != current_app_version:
```
Comment (barredterra) [inline banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py]: - Shouldn't this be `<`? (convert `current_version` to a `Version` object first)
- The latest release should probably be specific to the current branch. (A v15 release won't be of much use for a v14 user -- applies to ERPNext as well, btw)

Comment (marination) [reply banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py]: > applies to ERPNext as well, btw

Do you suggest i do this check for erpnext too ?

Comment (barredterra) [reply banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py]: If it's easier to fix there, it wouldn't hurt :D

Comment (marination) [reply banking/klarna_kosma_integration/doctype/banking_settings/banking_settings.py]: An issue could be due to frappe being outdated too. Let's stick to banking. I think we're entering devops territory 😄

**Pull request merged**